utils/helpers.ts

```
@@ -0,0 +1,404 @@
+}
+
+export const isLessThanFiveDays = (date: Date): boolean => {
+  const currentDate = new Date();
+  const timeDiff = currentDate.getTime() - date.getTime();
+  const dayDiff = timeDiff / (1000 * 3600 * 24);
+  return Math.floor(Math.abs(dayDiff)) < 5;
+};
+
+export const sanitize = (url: string): string => {
+  const accents = [
+    /[\u0300-\u030f]/g,
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)